project_timeline/hooks.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Open Source Integrators - Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def populate_date_start(cr, registry):
```

Comment:
This should be moved to "migrations"

Reply:
What would the cons of having it as it is?

Reply:
What happens if you already have it installed, there is no migration script that will execute this.

Reply:
Exactly. It should be as hook, but also call it from a migration script:

```python
from odoo.addons.project_timeline.hooks import populate_date_start
from odoo.modules import registry


def migrate(cr, version):
    populate_date_start(cr, registry)
```